h/models/token.py

```diff
@@ -24,6 +24,9 @@ class Token(Base, mixins.Timestamps):
     #: to, for example, one of the short-lived JWTs that the client uses).
     prefix = u'6879-'
 
+    #: A prefix that identifies a token as a refresh token.
+    refresh_token_prefix = u'7980-'
```

Comment:
Hmm, I don't think that `binascii.hexlify(os.urandom(16))` will ever return a string with a dash?